pkgs/applications/virtualization/sail-riscv/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+
+, arch
+
+, ocamlPackages
+, ocaml
+, zlib
+, z3
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "sail-riscv";
+  version = "0.5";
+
+  nativeBuildInputs = with ocamlPackages; [ ocamlbuild findlib ocaml z3 ];
+  buildInputs = with ocamlPackages; [ sail zlib ];
+
+  prePatch = "rm -r prover_snapshots";
```

Multiline strings are more readable.
```suggestion
  prePatch = ''
    rm -r prover_snapshots
  '';
```